specification/mysql/resource-manager/Microsoft.DBforMySQL/Backups/preview/2022-09-30-preview/BackupAndExport.json

```diff
@@ -0,0 +1,374 @@
+          "BackupAndExport"
+        ],
+        "operationId": "BackupAndExport_Create",
+        "x-ms-examples": {
+          "Create and Export Backup": {
+            "$ref": "./examples/BackupAndExport.json"
+          }
+        },
+        "description": "Exports the backup of the given server by creating a backup if not existing.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common-types/v1/common-types.json#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BackupAndExportRequest"
+            },
+            "description": "The required parameters for creating and exporting backup of the given server."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/BackupAndExportResponse"
+            }
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Location": {
+                "description": "URL to retrieve the final result after operation completes.",
+                "type": "string"
+              },
+              "Azure-AsyncOperation": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforMySQL/flexibleServers/{serverName}/validateBackup": {
+      "post": {
+        "tags": [
+          "BackupAndExport"
+        ],
+        "operationId": "BackupAndExport_ValidateBackup",
+        "x-ms-examples": {
+          "Validate Backup": {
+            "$ref": "./examples/ValidateBackup.json"
+          }
+        },
+        "description": "Validates if backup can be performed for given server.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
```

Comment:
>v2 [](http://example.com/codeflow?start=72&length=2)

use the latest relevant version

Reply:
ApiVersionParameter and ResourceGroupNameParameter will use v5, but other definition will use v2, in our e2e test, we still use a lower version autorest to generate SDK which does not support "format"